server/performance.py

```diff
@@ -0,0 +1,92 @@
+from typing import TYPE_CHECKING
+from typing import TypedDict
+
+from akatsuki_pp_py import Beatmap as CalculatorBeatmap
+from akatsuki_pp_py import Calculator
+
+if TYPE_CHECKING:
+    from server.repositories.beatmaps import Beatmap
+    from server.repositories.scores import Score
+
```

Comment:
```suggestion

# TODO: we can improve the typing here to avoid the optionals
# https://github.com/cmyui/osu-server-professing/pull/33/files#r1181634986
```